extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerSetLayerStyleCommand.java

```diff
@@ -43,18 +46,26 @@
 	private String styleName = null;
 
 	@Parameter(description = "<layer name>")
+	@RestParameters(names = {
+		"layerName"
+	})
 	private List<String> parameters = new ArrayList<String>();
 	private String layerName = null;
 
 	@Override
 	public boolean prepare(
 			OperationParams params ) {
-		super.prepare(params);
```

Comment:
`super.prepare(params);` got eliminated, it seems like it needs to be there (at least for the CLI to work the same, presumably the services should be similar too).